packages/core/src/components/menu/menuItem.tsx

```diff
@@ -22,10 +22,32 @@ export interface IMenuItemProps extends IActionProps, ILinkProps {
     /** Item text, required for usability. */
     text: string;
 
+    /**
+     * Children of this component will be rendered in a __submenu__ that appears when hovering or
+     * clicking on this menu item. You can instead pass an array of props objects to the `submenu` prop.
```

Comment:
I still don't understand this second sentence. Instead of what?

Reply:
`submenu` array instead of `children` elements. how can i clarify?

Reply:
Do you mean to say that a user can render a submenu with __either__ children or the `submenu` prop? (the "instead" wording works counter to this, it suggests that only one option works)

Why do we want to support that?

Reply:
yes that's what I mean to say. this has been supported forever.

remember when I asked about making `MenuItem` children into the content (`text`) of the item and relegating submenu to props only?

Reply:
🔥  proposal: `submenu?: React.ReactNode; submenuProps?: IMenuItemProps[]`

Reply:
I'd rather not break the API here and only support one usage pattern for submenus

Reply:
> one usage pattern

but we already support two patterns, and have for a _very_ long time

> not break the API

removing one usage pattern would be an API break...

what should I do here?

Reply:
a cursory search _suggests_ no one is using the `submenu` prop, but hard to verify.

Reply:
ohh ok I thought this was adding new support. in that case I'd suggest removing the `submenu` prop while we can in 2.0